blog/index.html

```diff
@@ -0,0 +1,112 @@
+<!DOCTYPE html>
+<html lang="ja">
+  <head>
+    <meta name="viewport" content="width=device-width,initial-scale=1" />
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-Uh-Compatible" content="ie=edge" />
+    <link rel="stylesheet" href="/style.scss" />
+    <link rel="stylesheet" href="/blog/style.scss" />
+    <script
+      src="https://code.jquery.com/jquery-3.5.0.min.js"
+      integrity="sha256-xNzN2a4ltkB44Mc/Jz3pT4iU1cmeR0FkXs4pru/JxaQ="
+      crossorigin="anonymous"
+    ></script>
+    <script src="/contact/contact-script.js"></script>
+    <title>Document</title>
+  </head>
+  <body>
+    <div class="global-wrapper">
+    <header class="global-header">
+      <h1 class="global-logo">
+        <img src="/img/logo-knicka-wide.png" alt="knicka office" />
+      </h1>
+      <nav class="global-nav">
+        <label for="GLOBAL_NAV_TOGGLE_BUTTON" class="global-nav__toggle-button">
+          <div class="global-nav__button-line parts-toggle-munu">
+            <span></span><span></span><span></span>
+          </div>
+        </label>
+        <input
+          type="checkbox"
+          id="GLOBAL_NAV_TOGGLE_BUTTON"
+          class="common-display-toggle-check"
+        />
+        <ul class="global-nav__menu common-nav-list">
+          <li class="global-nav__item common-nav-list__item">
+            <a href=""><span>ABOUT</span><span>オフィスのこと</span></a>
+          </li>
+          <li class="global-nav__item common-nav-list__item">
+            <a href=""><span>CLIENT</span><span>顧客</span></a>
+          </li>
+          <li class="global-nav__item common-nav-list__item">
+            <a href=""><span>BLOG</span><span>ブログ</span></a>
+          </li>
+          <li class="global-nav__item common-nav-list__item">
+            <a href=""><span>CONTACT</span><span>お問い合わせ</span></a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main class="main-content-wrapper">
+    <article class="blog-main">
+        <div class="blogHeaderNav">
+            <button class="blogHeaderNav__day-before">

```

Comment:
```suggestion
        <div class="blog-header-nav">
            <button class="blog-header-nav__day-before">
```
class名は、キャメルケースでは無く、ハイフンケースが良いです。